### PR TITLE
Last updates before release

### PR DESCRIPTION
**Pull request by s0undt3ch**
### What does this PR do?
See title

### CODE REVIEW

**Review by krionbsd — APPROVED** [review]

**Comment by s0undt3ch** [comment]
Arch Linux builds will fail :/

https://bugs.archlinux.org/task/69563

**Comment by s0undt3ch** [comment]
openSUSE tumbleweed also fails currently, @myii got a clue as to why?

File: ChangeLog [hunk 1]
```diff
@@ -1,5 +1,17 @@
 Version TBD (In Progress on the Develop Branch):
 
+Version 2020.10.20:
```
Comment (myii) [inline ChangeLog:3]:
```suggestion
Version 2021.03.02:
```

Comment (s0undt3ch) [reply ChangeLog:3]: Dam, I though I had updated that ....

**Comment by s0undt3ch** [comment]
@myii about tumbleweed, any clue?

File: ChangeLog (hunk 1 above)
Comment (s0undt3ch) [reply ChangeLog:3]: Fixed in https://github.com/saltstack/salt-bootstrap/pull/1542

**Comment by myii** [comment]
> openSUSE tumbleweed also fails currently, @myii got a clue as to why?

Started failing ever since `openssh-server-8.4p1-4.1.x86_64` was released, haven't had a chance to investigate further.  I'll try to have a look and see if there's a way around it.

In terms of Arch Linux, got a message in the `#irc` channel recently about the latest version being installable directly using `pacman -S salt`:

* https://freenode.logbot.info/salt/20210227#c7060250
* https://archlinux.org/packages/community/any/salt/

**Comment by myii** [comment]
> @myii about tumbleweed, any clue?

@s0undt3ch Got some interesting feedback for you.  Tried some stuff out earlier today but needed a moment to collect the findings all together.

---

#### Outline of the situation

https://forums.opensuse.org/showthread.php/550195-Install-git-in-docker

> ```
> docker run ... --security-opt seccomp:unconfined ...
> ```
>
> In the light of this information, it is not really an openSUSE/TW issue, more due to the way that openssh-server is being packaged now in TW and the way docker is packaged on Debian/Ubuntu.

https://www.reddit.com/r/openSUSE/comments/lm48zo/install_git_in_docker_tumbleweed/

> Known bug in old docker/libseccomp. You have to upgrade. Or use `--security-opt seccomp:unconfined`.

---

#### Current configuration is working in GitLab CI

This was the actually the last thing I tested but the most worthwhile finding.  I set up GitLab CI/CD mirroring of my `salt-bootstrap` fork as we do for CI across SaltStack Formulas.  Then pushed through a minimal `.gitlab-ci.yml`.  For easy comparison, I adjusted the GitHub Actions to only run for Tumbleweed as well.

This is the commit: https://github.com/myii/salt-bootstrap/commit/662751174912f84d9e81bcc8ccdd3da3bcf10e7c.

These are the results:

* https://github.com/myii/salt-bootstrap/actions/runs/615156291 -- failure
* https://gitlab.com/myii/salt-bootstrap/-/pipelines/264312416 -- success

---

#### Setting `--security-opt seccomp:unconfined` for `docker build`

We've been suffering the same Tumbleweed failures in our `salt-image-builder` repo as well.  The one difference between the pipeline above and the `salt-image-builder` repo is that they are using different `dind` (Docker-in-Docker) images:

* Above is using a custom Ubuntu 18.04: https://gitlab.com/myii/salt-image-builder/-/blob/feat/dockerfiles-for-ssf-gitlab-ci/Dockerfile.dind-ruby-bionic#L1.
* `salt-image-builder` is using Alpine: https://gitlab.com/myii/salt-image-builder/-/blob/feat/dockerfiles-for-ssf-gitlab-ci/Dockerfile.dind-ruby#L1.

The `--security-opt seccomp:unconfined` actually worked and we're producing Tumbleweed pre-salted images again:

* https://gitlab.com/saltstack-formulas/infrastructure/salt-image-builder/-/merge_requests/86

Thought I'd try that here but while the option is definitely being picked up, it's still failing.

This is the commit: https://github.com/myii/salt-bootstrap/commit/a5e82ca09690b13919192776331b030ac031a21d.

These are the results:

* https://github.com/myii/salt-bootstrap/actions/runs/615195628 -- failure
  - https://github.com/myii/salt-bootstrap/runs/2016225515?check_suite_focus=true#step:8:35
  - `docker_command: docker -H unix:///var/run/docker.sock build --build-arg=security-opt='seccomp=unconfined' -f Dockerfile-kitchen20210302-3062-dvumgg . shell_opts: {...}`
* https://gitlab.com/myii/salt-bootstrap/-/pipelines/264317272 -- success
  - Not necessary here, since already succeeding without.

---

Hopefully, there's enough info above to help figure out what needs to be done on this side.  I haven't spent much time with GitHub Actions in order to provide any useful suggestions, other than looking at options relevant to `runs-on: ubuntu-latest`.

**Pull request merged**